docs/site/themes/template/layouts/_default/resources.html

```diff
@@ -121,7 +121,7 @@ <h2>Additional resources</h2>
                         <div class="panel" id="panelAdditional3" aria-labelledby="additional3" role="region">
                             <br />
                             <div class="subpanel">
-                                <h3>Enlightening</h3>
+                                <h3>Enlightning</h3>
```

Comment:
The current spelling is actually correct.

https://www.merriam-webster.com/dictionary/enlightening